staging/vhost-device-sound/src/device.rs

```diff
@@ -678,7 +684,7 @@ impl VhostUserSoundBackend {
     }
 }
 
-impl VhostUserBackend<VringRwLock, ()> for VhostUserSoundBackend {
+impl VhostUserBackendMut<VringRwLock, ()> for VhostUserSoundBackend {
```

Comment:
Why this change? I'd put in a separate commit with a description.

Reply:
> Looks like it's from the dependency updates.
> 
> @dorindabassey my suggestion is first update the dependencies in a standalone commit, and add the test changes in a second one.

Yes, please ;-)

Reply:
looks like this change is not necessary. I changed it because of this [test](https://github.com/rust-vmm/vhost-device/pull/489/files/d7179cc9091887f20caebf1c810332a87769108e#diff-ab6dc4ed31cc538191be45fbe8e1bce7df597f769bea01d2390a96faeffa0700R351)
afterall I can do `assert_eq!(backend.threads.len(), 1);`

I will split the commits